apps/fishing-map/features/user/user.slice.ts

```diff
@@ -30,19 +30,21 @@ export const GFW_DEV_GROUP_ID = 'development-group'
 export const ADMIN_GROUP_ID = 'admin-group'
 export const DEFAULT_GROUP_ID = 'Default'
 export const PRIVATE_SUPPORTED_GROUPS = [
-  'Indonesia',
-  'Peru',
-  'Panama',
+  'Belize',
   'Brazil',
-  'Mexico',
-  'Ecuador',
   'Costa_Rica',
-  'Belize',
+  'Ecuador',
+  'Indonesia',
+  'Mexico',
+  'Panama',
+  'Papua New Guinea',
+  'Peru',
   'SSF-Aruna',
-  'SSF-Rare',
   'SSF-Ipnlf',
+  'SSF-Rare',
 ]
 export const USER_GROUP_WORKSPACE = {
+  'papua new guinea': 'papua_new_guinea',
```

Comment:
Any reason for this group to not follow the convention of using `-` ?

Reply:
Good catch, fixed now!